docarray/documents/video.py

```diff
@@ -97,12 +99,30 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[VideoUrl]
-    audio: Optional[AudioDoc] = AudioDoc()
-    tensor: Optional[VideoTensor]
-    key_frame_indices: Optional[AnyTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[VideoBytes]
+    url: Optional[VideoUrl] = Field(
+        description='URL to the remote video file that needs to be loaded',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/mov_bbb.mp4?raw=true',
+    )
+    audio: Optional[AudioDoc] = Field(
+        description='Audio document document associated with the video',
+        default=AudioDoc(),
+    )
+    tensor: Optional[VideoTensor] = Field(
+        description='''Tensor object of the video which be specified to one of
+        `VideoNdArray`, `VideoTorchTensor`, `VideoTensorFlowTensor`''',
+    )
+    key_frame_indices: Optional[AnyTensor] = Field(
+        description='List of all the key frames in the video',
+        example='np.arrange(101)',
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='''Embedding field is used to store tensor objects of type
+        Tensorflow, PyTorch, NumPy and Jax''',
+        example='np.zeros((3, 32, 32))',
+    )
+    bytes_: Optional[VideoBytes] = Field(
```

Comment:
Skipping example here since we will need to do get bytes from video url and needs a multi-line example